.github/workflows/release.yml

```diff
@@ -130,6 +135,45 @@ jobs:
               --docker-password $DOCKERHUB_TOKEN \
               --force \
               --branch $TARGET_BRANCH
+      - name: Run customer 1 workflows
+        if: ${{ github.event.inputs.should-run-customer-tests == 'true' }}
+        env:
+          CUSTOMER1_AUTH_TOKEN: ${{ secrets.CUSTOMER1_TOKEN }}
```

Comment:
Did we check with the relevant people to make sure it's ok to have this in GHA?

How do we plan to generate this token -- a limited-access token from one of our accounts that is in the org that has access to the projects?

Where will we track (internally) which customer / project is "customer1" "customer2" etc. and for which projects the customer token needs CONTRIBUTE, VIEW, etc. for when we have to re-generate the tokens in the future? @jdidion  @commandlinegirl

Reply:
All good questions. I'm open to suggestions. My proposal is to manage it in a Google sheet. For tokens, either the customer can add us to their project with proper permissions and we can generate the scoped token, or they can generate the token and share it with us securely. The former is preferable so we don't have to bug the customer every time the token needs to be refreshed.